builtin/providers/azurerm/provider.go

```diff
@@ -85,7 +85,8 @@ func Provider() terraform.ResourceProvider {
 			"azurerm_lb_probe":                resourceArmLoadBalancerProbe(),
 			"azurerm_lb_rule":                 resourceArmLoadBalancerRule(),
 
-			"azurerm_managed_disk": resourceArmManagedDisk(),
+			"azurerm_managed_disk":  resourceArmManagedDisk(),
+			"azurerm_managed_image": resourceArmManagedImage(),
```

Comment:
After our discussion, I changed this to be "azurerm_image" instead of "azurerm_managed_image" - I will communicate the possible confusion over already existing "image_uri" to the docs team.